Barlist/Report.h

```diff
@@ -32,7 +32,13 @@ class CReport
 
    static CString GetReportHeader();
 
-   void BuildReport(IBarlist* pBarlist);
+   enum ReportOptions {
+	   REPORT_TOTAL_QUANTITIES = 1,
+	   REPORT_GROUP_QUANTITIES = 2,
+	   REPORT_TOTAL_AND_GROUP_QUANTITIES = 3
+   };
```

Comment:
consider changing this to a 'enum class' because it is more type-safe.

Also, the enum is setup for three options, but there are only two implemented. Total quantities and Total+Group Quantities.

Reply:
Assuming you mean to change from 'enum' to 'enum' class. Done. Yes, I only see 2 options implemented. Done